### PR TITLE
JSON output format and other fixes

### PR DESCRIPTION
**Pull request by unparalleled-js**
* Use `json.dumps()` rather than `pprint` to follow suit with other `show` commands' output format.
* Impove doc strings language
* Change default param from `"."` to `os.getwd()`

### CODE REVIEW

**Comment by unparalleled-js** [comment]
I will put any remaining changes on this pr ... standby

File: src/code42cli/cmds/cases.py [hunk 1]
```diff
@@ -20,20 +19,23 @@
...
     "--case-number", type=int, help="The number assigned to the case.", required=True
 )
-name_option = click.option("--name", help="Name of the case.",)
-assignee_option = click.option("--assignee", help="User UID of the assignee.")
-description_option = click.option("--description", help="Description of the case.")
-findings_option = click.option("--findings", help="Findings on the case.")
-subject_option = click.option("--subject", help="User UID of a subject of the case.")
+name_option = click.option("--name", help="The name of the case.",)
+assignee_option = click.option(
+    "--assignee", help="The UID of the user to assign to the case."
```
Comment (unparalleled-js) [inline src/code42cli/cmds/cases.py:24]: `to assign to the case` makes sense for both `update` and `create`, where this is used.

**Comment by unparalleled-js** [comment]
OOOk I am done now

**Comment by unparalleled-js** [comment]
My final concern is that we have a --change-device-name flag for deactivating devices that prepends deactivated_<todays-date> to the name of the device... However, there is no way to undo this with the CLI... We can reactivate the device but we can't rename the device or remove the deactivate prefix..... Not sure what to do here. The flag in general seems like a really specific use-case, maybe too specific for the CLI.

**Review by alanag13 — APPROVED** [review]

**Pull request merged**